### DISCUSSION TITLE
Verifying step 3 microSD Card Socket (SMD)

### DISCUSSION THREAD

**Discussion by Yorxs** [thehighnibble/thehighnibble.github.io#156] [IMSAI8080esp replica]
I'm verifying that none of the pins for the uSD Card are bridged to the metal casing. When I probe the metal casing and uSD pin 6, I get continuity between the two.  Would someone be able to verify for me if that is normal?

**Comment by dmcnaugh** [comment]
Yes, pin 6 is Ground (GND) and the metal casing is grounded also.

**Comment by Yorxs** [comment]
Thank you for verifying for me.
